src/main/java/model/MoveCarStrategy.java

```diff
@@ -0,0 +1,24 @@
+package model;
+
+public class MoveCarStrategy {
+    private static final int MOVE_THRESHOLD = 4;
+    private static final int RANDOM_NUMBER_BOUND = 10;
+    private final NumberGenerator numberGenerator;
```

Comment:
```suggestion
public class MoveCarStrategy {

    private static final int MOVE_THRESHOLD = 4;
    private static final int RANDOM_NUMBER_BOUND = 10;

    private final NumberGenerator numberGenerator;
```

사소한 부분이긴한데 이렇게 줄바꿈으로 구역을 나눠주시면 더 좋아요
코드는 사소한 규칙을 모아서 티끌 모아 태산을 만든다고 보시면 됩니다

Reply:
이 부분까지 신경쓰지 못한 것 같습니다. 줄바꿈 규칙에 대해서도 공부해보아야겠다는 생각이 드네요!